src/machine/enums.ts

```diff
@@ -20,15 +20,15 @@ enum CompoundStates {
 }
 
 enum ClassificationEnum {
```

Comment:
Should we use only one Enum to avoid updating it in 2 different places?

Reply:
i can't do it, since i want to use in server folder and in src folder